src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/UsageBasedMetadataManager.cs

```diff
@@ -122,6 +120,22 @@ protected override void Graph_NewMarkedNode(DependencyNodeCore<NodeFactory> obj)
             {
                 _customAttributesWithMetadata.Add(customAttributeMetadataNode.CustomAttribute);
             }
+
+            var reflectableFieldNode = obj as ReflectableFieldNode;
+            if (reflectableFieldNode != null)
```

Comment:
I know this is based on the existing code, but it would look "nicer" like this:
```C#
if (obj is ReflectableFieldNode reflectableFieldNode)
```

Maybe even change the whole thing into a switch?

Reply:
I tend to follow the existing style in the file and don't change surrounding style as part of unrelated changes. It makes history/git blame tidier. We could reformat in a single pass if it's a problem.